search-app/docs/api.md

```diff
@@ -246,7 +245,7 @@ The CMR contains many more results than can be returned in a single response so
 
 `offset` is a 0 based index into the result set of a query. If a search matched 50 items the parameters `offset=3&page_size=5` would return 4th result through the 8th result.
 
-You can not page past the 1 millionth item. Please contact the CMR Team through the [CMR Client Developer Forum](https://wiki.earthdata.nasa.gov/display/CMR/CMR+Client+Developer+Forum) if you need to retrieve items in excess of 1 million from the CMR. Additionally granule queries which do not target a set of collections are limited to paging up to the 10000th item.
+You can not page past the 1 millionth item. Please contact the CMR Team at cmr-support@earthdata.nasa.gov if you need to retrieve items in excess of 1 million from the CMR. Additionally granule queries which do not target a set of collections are limited to paging up to the 10000th item.
```

Comment:
I think the support email should be 'support@earthdata.nasa.gov'

Reply:
Either way is fine. cmr-support will come directly to the CMR Kayako bucket; support will go to the general Kayako ticket bucket, and then it will be forwarded to CMR by the Kayako gate-keepers. Since this is specific to CMR support, I think cmr-support is reasonable.